src/Jackett/Indexers/NxtGn.cs

```diff
@@ -0,0 +1,185 @@
+            AddCategoryMapping(31, TorznabCatType.TVHD);
+            AddCategoryMapping(21, TorznabCatType.TVFOREIGN);
+            AddCategoryMapping(46, TorznabCatType.TV);
+            AddCategoryMapping(46, TorznabCatType.TVHD);
+            AddCategoryMapping(45, TorznabCatType.TV);
+            AddCategoryMapping(45, TorznabCatType.TVSD);
+            AddCategoryMapping(24, TorznabCatType.TVFOREIGN);
+            AddCategoryMapping(26, TorznabCatType.TV);
+            AddCategoryMapping(26, TorznabCatType.TVHD);
+            AddCategoryMapping(26, TorznabCatType.TVWEBDL);
+            AddCategoryMapping(33, TorznabCatType.MoviesHD);
+            AddCategoryMapping(33, TorznabCatType.Movies);
+            AddCategoryMapping(17, TorznabCatType.MoviesForeign);
+            AddCategoryMapping(17, TorznabCatType.MoviesDVD);
+            AddCategoryMapping(9, TorznabCatType.MoviesHD);
+            AddCategoryMapping(9, TorznabCatType.Movies);
+            AddCategoryMapping(9, TorznabCatType.MoviesBluRay);
+            AddCategoryMapping(43, TorznabCatType.TV);
+            AddCategoryMapping(43, TorznabCatType.TVHD);
+            AddCategoryMapping(43, TorznabCatType.TVWEBDL);
+        }
+
+        public async Task ApplyConfiguration(JToken configJson)
+        {
+            var loginPage = await RequestStringWithCookies(LoginUrl);
+            CQ loginDom = loginPage.Content;
+            var loginPostUrl = loginDom["#login"].Attr("action");
+
+            configData.LoadValuesFromJson(configJson);
+            var pairs = new Dictionary<string, string> {
+                { "username", configData.Username.Value },
+                { "password", configData.Password.Value }
+            };
+            // Get inital cookies
+            CookieHeader = string.Empty;
+            var response = await RequestLoginAndFollowRedirect(SiteLink + loginPostUrl, pairs, CookieHeader, true, null, LoginUrl);
+
+            await ConfigureIfOK(response.Cookies, response.Content != null && response.Content.Contains("Velkommen tilbage"), () =>
+            {
+                CQ dom = response.Content;
+                var messageEl = dom["inputs"];
+                var errorMessage = messageEl.Text().Trim();
+                throw new ExceptionWithConfigData(errorMessage, configData);
+            });
+
+            var profilePage = await RequestStringWithCookies(ProfileUrl, response.Cookies);
+            CQ profileDom = profilePage.Content;
+            var passKey = profileDom["input[name=resetkey]"].Parent().Text();
+            passKey = passKey.Substring(0, passKey.IndexOf(' '));
+            configData.RSSKey.Value = passKey;
+            SaveConfig();
+        }
+
+        public async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+            var breakWhile = false;
+            var page = 0;
+            while (true)
```

Comment:
Can you make this a for loop any only allow three pages? A little concerned you could search for 2015 etc and it'll hit the server who knows how many times?